src/test/java/uk/gov/hmcts/reform/draftstore/service/draftservice/ReadSingleTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.draftstore.service.draftservice;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.draftstore.exception.NoDraftFoundException;
+import uk.gov.hmcts.reform.draftstore.service.UserAndService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReadSingleTest extends BaseTest {
```

Comment:
We could have one more test checking what we return when the draft exists and is requested for the right (user, service) pair.

Reply:
added extra tests